pkg/scheduler/apis/config/types.go

```diff
@@ -270,3 +271,53 @@ func (p *Plugins) Append(src *Plugins) {
 	p.PostBind = appendPluginSet(p.PostBind, src.PostBind)
 	p.Unreserve = appendPluginSet(p.Unreserve, src.Unreserve)
 }
+
+// Apply merges custom plugins into default plugins.
```

Comment:
we can be more explicit here. Something like:
Apply merges the plugin configuration from custom plugins, handling disabled sets.